tests/unit/test_settings.py

```diff
@@ -90,7 +90,18 @@ def test_is_supported_filetype_fifo(self, tmpdir):
     def test_src_paths_are_combined_and_deduplicated(self):
         src_paths = ["src", "tests"]
         src_full_paths = (Path(os.getcwd()) / f for f in src_paths)
-        assert Config(src_paths=src_paths * 2).src_paths == tuple(src_full_paths)
+        assert sorted(Config(src_paths=src_paths * 2).src_paths) == sorted(src_full_paths)
```

Comment:
I assumed the order of `src_paths` does not matter... does it? :eyes:

Reply:
It should not!